<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0GMBHRP" ignored="0" name="BankingApp Test(failed)" tests="4" failures="4" timestamp="23 Feb 2022 12:08:16 GMT" time="215.582" errors="0">
  <testcase name="loginDDT" time="30.947" classname="com.sf.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.sf.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="15.409" classname="com.sf.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.sf.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="17.308" classname="com.sf.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.sf.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginTest" time="42.46" classname="com.sf.testCases.TC_LoginTest_001">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.sf.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:38)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- BankingApp Test(failed) -->
